internal/archive/mta_location_test.go

```diff
@@ -59,22 +59,33 @@ var _ = Describe("Path", func() {
 		Ω(location.GetMtaYamlPath()).Should(Equal(getPath("mta.yaml")))
 	})
 	It("GetSourceModuleArtifactRelPath - artifact is file", func() {
-		location := Loc{SourcePath: getPath()}
-		Ω(location.GetSourceModuleArtifactRelPath("m1", filepath.Join(getPath(), "m1", "folder", "abc.jar"), false)).
-			Should(Equal(string(filepath.Separator) + "folder"))
+		location := Loc{SourcePath: getPath("testdata", "mtahtml5")}
+		relPath, err := location.GetSourceModuleArtifactRelPath("ui5app", getPath("testdata", "mtahtml5", "ui5app", "webapp", "Component.js"))
```

Comment:
Does it matter if the file is an archive (e.g. jar like before you changed the test)?

Reply:
no, but now existence check added and jar does not exist

Reply:
Maybe there should be a test for an archive too. You can add one...

Reply:
added...